wrappers/ios/Sources/Wrapper/Writer/ZXIBarcodeWriter.h

```diff
@@ -14,6 +14,13 @@ NS_ASSUME_NONNULL_BEGIN
                      height:(int)height
                      format:(ZXIFormat)format
                       error:(NSError **)error;
+
+-(nullable CGImageRef)writeData:(NSData *)data
```

Comment:
I'm not really happy with the name `writeData`. First of all: am I right in assuming that ObjectiveC does not support function overloading (based on the type of parameter), so this function has to have a different name than the existing one? The first best alternative that comes to my mind is `writeBytes` (see `class Result`). It would then make a lot of sens to rename the other to `writeText`.

The next question would be: how disruptive would it be to later change the name / parameter list of this function. I'm asking because I'm working on a new `WriteBarcode` API that is supposed to be analogous to the `ReadBarcode` API, meaning it would get passed an `EncodeHints` struct, similar to `DecodeHints`. Introducing something new now that effectively prohibits me from doing 'the right thing' later is what I'd like to avoid.